README.md

```diff
@@ -8,13 +8,15 @@ Provides Promises/A+ compliant versions of all your favorite AWS SDK clients.
 
 #### Disclaimer
 
-This module is v0.0.1 and is a work in progress. Not all of the AWS SDK clients are wrapped
+This module is v0.2.0 and is a work in progress. Not all of the AWS SDK clients are wrapped
```

Comment:
Thank you for doing that. It occurred to me that we could probably just remove that bit with the version number so we don't have to keep updating it.

Could you change this to:

```
"This module is a work in progress"
```

no version number needed.